From 3cbd0f26b644e680a10efa0ae4e5391aedbe583d Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Wed, 21 Jun 2023 10:04:45 +0200
Subject: [PATCH 309/954] feat(stm32mp2-fdts): update compatible string for
 watchdog

The compatible string for STM32MP25x watchdog is the same as for
the STM32MP1xx devices.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 5cbdc3821691d4a5b6d4db8c2562f7b98ea97388 feat(stm32mp2-fdts): update compatible string for watchdog]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I57c80729dfd486731efd4ad1ed9ff0334dcfa5a9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/314038
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp251.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index cb8da66223..03bd77df19 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -265,7 +265,7 @@
 			};
 
 			iwdg1: watchdog@44010000 {
-				compatible = "st,stm32mp2-iwdg";
+				compatible = "st,stm32mp1-iwdg";
 				reg = <0x44010000 0x400>;
 				clocks = <&rcc CK_BUS_IWDG1>, <&rcc LSI_CK>;
 				clock-names = "pclk", "lsi";
-- 
2.53.0

